lib/components/user/user-account-screen.js

```diff
@@ -59,6 +106,81 @@ class UserAccountScreen extends Component {
     this._handleExit()
   }
 
+  /**
+   * Reverts the user phone number and verification state in the database.
+   */
+  _handleRevertPhoneNumber = async () => {
```

Comment:
Please remove all revert code. I believe it is not needed for a change that needs confirmation.

Reply:
Updated